spec/spec.md

```diff
@@ -508,20 +513,20 @@ get_capabilities
 ```
 {
     "supports_clone_authorization": <supports_clone_authorization>,
-    "supports_sign_and_send_transactions": <supports_sign_and_send_transactions>,
     "max_transactions_per_request": <max_transactions_per_request>,
     "max_messages_per_request": <max_messages_per_request>,
     "supported_transaction_versions": [<supported_transaction_versions>, ...]
+    "features": ["<feature_id>"]
 }
 ```
 
 where:
 
 - `supports_clone_authorization`: `true` if the [`clone_authorization`](#clone_authorization) method is supported, otherwise `false`
-- `supports_sign_and_send_transactions`: `true` if the [`sign_and_send_transactions`](#sign_and_send_transactions) method is supported, otherwise `false`
 - `max_transactions_per_request`: (optional) if present, the max number of transaction payloads which can be signed by a single [`sign_transactions`](#sign_transactions) or [`sign_and_send_transactions`](#sign_and_send_transactions) request. If absent, the implementation doesn't publish a specific limit for this parameter.
 - `max_messages_per_request`: (optional) if present, the max number of transaction payloads which can be signed by a single [`sign_messages`](#sign_messages) request. If absent, the implementation doesn't publish a specific limit for this parameter.
 - `supported_transaction_versions`: the Solana network transaction formats supported by this wallet endpoint. Allowed values are those defined for [`TransactionVersion`](https://solana-labs.github.io/solana-web3.js/modules.html#TransactionVersion) (for e.g., `"legacy"`, `0`, etc).
+- `features`: a list of [feature identifiers](#feature-identifiers) for the optional features supported by this wallet endpoint.
```

Comment:
> optional features

Does this mean that the `solana:` namespaced features won't be in this list?

Reply:
that was my thought, yes. that mandatory features would not be listed here. 

should mandatory features have a specific namespace? or do you think explicitly calling out mandatory vs optional features is sufficient?

Reply:
I agree, I don't think mandatory features need to be included in this list. If they are, you get into a situation where a mandatory feature is missing - in that case, what does a client do? Better if the mandatory features are assumed (based on the negotiated protocol version), since there is nothing a client can realistically do if they are missing.